pnpm-lock.yaml

```diff
@@ -16188,6 +16188,14 @@ packages:
     dependencies:
       glob-to-regexp: 0.4.1
       graceful-fs: 4.2.11
+    dev: false
+
+  /watchpack@2.4.1:
```

Comment:
This is wild, the `pnpm dedupe` seems to have done the _opposite_ effect somehow. There's now two versions of `watchpack` in the lockfile.

Reply:
wtf? any idea how to proceed?